docs/core/whats-new/dotnet-10/libraries.md

```diff
@@ -150,3 +150,68 @@ The new `ExportPkcs12` <!--TODO: add xref --> methods on <xref:System.Security.C
 which produces an output supported by almost every library/platform that supports reading PKCS#12/PFX by choosing an older encryption algorithm. `Pkcs12ExportPbeParameters.Pbes2Aes256Sha256` <!--TODO: add xref --> indicates that AES should be used instead of 3DES (and SHA-2-256 instead of SHA-1), but the output may not be understood by all readers (such as Windows XP).
 
 Callers who want even more control can instead utilize the overload that accepts a <xref:System.Security.Cryptography.PbeParameters>.
+
+## New AOT-safe constructor for `ValidationContext`
+
+The <xref:System.ComponentModel.DataAnnotations.ValidationContext> class, used during options validation, now includes a new constructor that explicitly accepts the `displayName` parameter. This ensures AOT safety, allowing its use in native builds without warnings:
+
+```csharp
+public sealed class ValidationContext
+{
+    public ValidationContext(object instance,
+        string displayName, 
+        IServiceProvider? serviceProvider = null, 
+        IDictionary<object, object?>? items = null);
+}
+```
+
+## Support for telemetry schema URLs in `ActivitySource` and `Meter`
+
+<xref:System.Diagnostics.ActivitySource> and <xref:System.Diagnostics.Metrics.Meter> now support specifying a telemetry schema URL during construction, aligning with OpenTelemetry specifications. This ensures consistency and compatibility for tracing and metrics data. Additionally, the update introduces `ActivitySourceOptions`, simplifying the creation of <xref:System.Diagnostics.ActivitySource> instances with multiple configuration options.
+
+```csharp
+public sealed partial class ActivitySource
+{
+    public ActivitySource(ActivitySourceOptions options);
+    public string? TelemetrySchemaUrl { get; }
+}
+
+public class ActivitySourceOptions
+{
+    public ActivitySourceOptions(string name);
+    public string Name { get; set; }
+    public string? Version { get; set; }
+    public IEnumerable<KeyValuePair<string, object?>>? Tags { get; set; }
+    public string? TelemetrySchemaUrl { get; set; }
+}
+
+public partial class Meter : IDisposable
+{
+    public string? TelemetrySchemaUrl { get; }
+}
+```
+
```

Comment:
Since the ML.NET updates aren't versioned with .NET 10, I'm not sure if it makes sense to put this and the next update in this article. For .NET 9, we decided to maintain a separate [What's new in ML.NET article](https://learn.microsoft.com/en-us/dotnet/machine-learning/whats-new/overview). If that article is no longer being updated, then we should probably delete it and combine the info into the existing What's new in .NET 9/10 articles.

Reply:
It looks like it hasn't been maintained since last year.

Reply:
Resolving this for now because I need to get this out the door today. We can figure out what to do with ML.NET before Preview 4 ships